lib/output.js

```diff
@@ -875,14 +881,19 @@ function OutputStream(options) {
         self.template_string.print(output);
     });
     DEFPRINT(AST_TemplateString, function(self, output) {
+        if (self.segments.length === 1 && typeof self.segments[0] === "string"
+            && output.option("quote_style") != 3 && !/"'/.test(self.segments[0])) {
+            return output.print_string(self.segments[0].replace(/\\`/g, "`"), '`', in_directive);
+        }
```

Comment:
This might need fine tuning, all it has to do is check if it's better to use string quotes over template quotes.